### ISSUE TITLE
[Bug] [Task Plugin] hive sql execute failed

### ISSUE THREAD

**Issue by SbloodyS** [apache/dolphinscheduler#7039]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

I got the following error when using sql node to execute hive sql
```
[ERROR] 2021-11-29 04:56:17.926 org.apache.dolphinscheduler.server.worker.runner.TaskExecuteThread:[210] - task scheduler failure
java.lang.NoClassDefFoundError: Could not initialize class org.apache.hadoop.conf.Configuration
        at org.apache.dolphinscheduler.plugin.task.util.CommonUtils.loadKerberosConf(CommonUtils.java:84)
        at org.apache.dolphinscheduler.plugin.task.datasource.hive.HiveDatasourceProcessor.getConnection(HiveDatasourceProcessor.java:129)
        at org.apache.dolphinscheduler.plugin.task.datasource.DatasourceUtil.getConnection(DatasourceUtil.java:59)
        at org.apache.dolphinscheduler.plugin.task.sql.SqlTask.executeFuncAndSql(SqlTask.java:184)
        at org.apache.dolphinscheduler.plugin.task.sql.SqlTask.handle(SqlTask.java:155)
        at org.apache.dolphinscheduler.server.worker.runner.TaskExecuteThread.run(TaskExecuteThread.java:192)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
```

### What you expected to happen

hive sql execute successfully.

### How to reproduce

using sql node to execute hive sql.

### Anything else

After copy lib/*.jar to lib/plugin/task/sql/,everything works fine.

### Version

2.0.0

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by calvinjiang** [comment]
Could you shed some more light on that? Give more details about this issue.

**Comment by lenboo** [comment]
> Could you shed some more light on that? Give more details about this issue.
@calvinjiang 
do you want fix this bug?

**Comment by SbloodyS** [comment]
![](https://files.catbox.moe/enymj8.png)

Running command like this in 2.0.0-release.This problem can be reproduced. @calvinjiang

**Comment by calvinjiang** [comment]
> ![](https://camo.githubusercontent.com/96871356123ff87425df494f88dd5460381c171b3b6f119234c97ce016867aab/68747470733a2f2f66696c65732e636174626f782e6d6f652f656e796d6a382e706e67)
> 
> Running command like this in 2.0.0-release.This problem can be reproduced. @calvinjiang

All right, I'll try to check this issue ASAP.

**Comment by calvinjiang** [comment]
> Could you shed some more light on that? Give more details about this issue.
> @calvinjiang
> do you want fix this bug?

Yea, I'll try to fix this issue.

**Comment by calvinjiang** [comment]
@SbloodyS This issue didn't reproduce in my environment. I guess that the issue may be fixed already. Could you pull or download new code  and then take a try one more time. If there will still be this issue , please tell me , I will have a look.

![image](https://user-images.githubusercontent.com/4928204/144010179-e26c047f-69df-4f8b-898c-a04433bf5876.png)

![image](https://user-images.githubusercontent.com/4928204/144010242-c3d19651-8967-4c1c-b722-69b48804ac41.png)

![image](https://user-images.githubusercontent.com/4928204/144010831-4cc2b120-df11-4f74-945c-87e242a7b4e2.png)

**Comment by SbloodyS** [comment]
I reinstall ds 2.0.0-release from https://www.apache.org/dyn/closer.lua/dolphinscheduler/2.0.0/apache-dolphinscheduler-2.0.0-bin.tar.gz.It can be reproduce every time.

```
[ERROR] 2021-11-30 16:34:34.938 org.apache.dolphinscheduler.server.worker.runner.TaskExecuteThread:[210] - task scheduler failure
java.lang.NoClassDefFoundError: org/apache/hadoop/conf/Configuration
        at org.apache.dolphinscheduler.plugin.task.util.CommonUtils.loadKerberosConf(CommonUtils.java:84)
        at org.apache.dolphinscheduler.plugin.task.datasource.hive.HiveDatasourceProcessor.getConnection(HiveDatasourceProcessor.java:129)
        at org.apache.dolphinscheduler.plugin.task.datasource.DatasourceUtil.getConnection(DatasourceUtil.java:59)
        at org.apache.dolphinscheduler.plugin.task.sql.SqlTask.executeFuncAndSql(SqlTask.java:184)
        at org.apache.dolphinscheduler.plugin.task.sql.SqlTask.handle(SqlTask.java:155)
        at org.apache.dolphinscheduler.server.worker.runner.TaskExecuteThread.run(TaskExecuteThread.java:192)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassNotFoundException: org.apache.hadoop.conf.Configuration
        at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
        at org.apache.dolphinscheduler.spi.plugin.DolphinPluginClassLoader.loadClass(DolphinPluginClassLoader.java:81)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
        ... 11 common frames omitted
```

There is no corresponding jar in lib/plugin/task/sql/

![](https://files.catbox.moe/xxl4p7.png)

**Comment by calvinjiang** [comment]
@SbloodyS Can you try to use the hive client command to access your hive under the same environment and system user like this:
```shell
./bin/beeline -u jdbc:hive2://xxxxxx:10000 -n your-name -p your-password
show databases;
````

**Comment by SbloodyS** [comment]
This is production environment.And show databases works fine in 1.3.9.@calvinjiang

**Comment by calvinjiang** [comment]
@SbloodyS I guess that the environment of the worker didn't find hadoop-common-xxx.jar that contains the class 'org.apache.hadoop.conf.Configuration'.

**Comment by SbloodyS** [comment]
@calvinjiang I think so. So I copy lib/*.jar to lib/plugin/task/sql. It works fine.

**Comment by zhongjiajie** [comment]
> @calvinjiang I think so. So I copy lib/*.jar to lib/plugin/task/sql. It works fine.

@SbloodyS  Does this issue fix by @calvinjiang suggest or not? If fix I'd like to close it.

**Comment by SbloodyS** [comment]
@zhongjiajie It's already fixed in 2.0.1-release.thanks.I will close it.

**Comment by wangw111** [comment]
@SbloodyS  which commit id for this issue
